From 8cafc68ef6a30c8cbef86f335f7375586129bf88 Mon Sep 17 00:00:00 2001
From: Richard Zhu <hongxing.zhu@nxp.com>
Date: Wed, 17 Oct 2018 11:09:16 +0800
Subject: [PATCH 4881/5242] MLK-19959 PCI: imx: make sure the sysfile api is
 add properly

commit  aa0aeff40198f956b855eec5c17f539e885a97c3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Make sure that the sysfile api of the pcie ep is add properly.

Signed-off-by: Richard Zhu <hongxing.zhu@nxp.com>
Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/pci/controller/dwc/pci-imx6.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pci/controller/dwc/pci-imx6.c b/drivers/pci/controller/dwc/pci-imx6.c
index eca22c7..7a40ed2 100644
--- a/drivers/pci/controller/dwc/pci-imx6.c
+++ b/drivers/pci/controller/dwc/pci-imx6.c
@@ -2570,6 +2570,7 @@ static int imx_pcie_probe(struct platform_device *pdev)
 		unsigned long timeout = jiffies + msecs_to_jiffies(300000);
 
 		/* add attributes for device */
+		imx_pcie_attrgroup.attrs = imx_pcie_ep_attrs;
 		ret = sysfs_create_group(&pdev->dev.kobj, &imx_pcie_attrgroup);
 		if (ret)
 			return -EINVAL;
-- 
1.7.9.5

